text/core_rtos.tex

```diff
@@ -23,7 +23,7 @@ \section{Starting the system with the loader}
 This, in turn, enables multi-stage boot where some functionality, such as attestation, secure key storage, and so on, are provided by a bootloader.
 These abstractions can all be built from capabilities and so, unlike systems based on protection rings such as TrustZone, an arbitrary number can be nested.
 
-If a compartment contains a global that is a pointer, initialised to point to another global, the loader will initialise the pointer by deriving a capability from one out of the compartment's code or data capabilities.
+If a compartment contains a global that is a pointer, initialised to point to another global, the loader will initialise the pointer by deriving a capability out of the compartment's code or data capabilities.
```

Comment:
I don't think this change is correct.  The compartment has two capabilities, code and data.  The new capability is derived from one of these.  Perhaps 'out of either...'?